ubuntudesign/vanillaframework.io
# Add docs theme

## Done
- Integrate docs-vanilla-theme 0.2.0
- Change mark-up to match the new theme

## QA
- Load vf.io
- Click through and check its styled with the docs theme

Fixes: https://github.com/ubuntudesign/vanillaframework.io/issues/17
